src/System.IO.FileSystem/tests/FileSystemTest.cs

```diff
@@ -22,22 +22,26 @@ protected static bool CanCreateSymbolicLinks
             get
             {
                 // Verify file symlink creation
-                var path = Path.GetTempFileName();
-                var linkPath = path + ".link";
-                var ret = MountHelper.CreateSymbolicLink(linkPath, path, isDirectory: false);
-                ret = ret && File.Exists(linkPath);
+                string path = Path.GetTempFileName();
+                string linkPath = path + ".link";
+                if (!MountHelper.CreateSymbolicLink(linkPath, path, isDirectory: false) || !File.Exists(linkPath))
```

Comment:
oh I see, I misunderstood your previous comment. In every case I can think of, subsequent Exists calls should always return true when CreateSymbolicLink succeeds.

I added the Exists check to better simulate what the actual tests will be doing; the're not going to just be creating links, theyll be operating on them, so it made sense to do the same in the conditional check in case there were a hypothetical situation where we could create a link but not access it. That is probably unnecessary though.